### DISCUSSION TITLE
[Announcement] Final exam

### DISCUSSION THREAD

**Discussion by goodtaeeun** [prosyslab-classroom/cs524-program-analysis#439] [Announcements]
(English version follows)

시간: 6월 10일 (월) 9:00 - 11:00 (120분)
장소: N1 102 (강의실과 동일)

- 오픈북 시험이지만 전자기기는 허용되지 않습니다.
- 모국어가 한국어인 한국학생들은 답안도 한국어로 작성해주시기 바랍니다.
- 시험 도중에는 문제에 대한 질문이 허용되지 않습니다. 적절한 가정이 필요한 경우 어떠한 가정하에 답변을 작성하였는지 명시해 주세요.
- 각 문항의 점수는 And 연산으로 채점됩니다. 즉, 답변의 내용이 모두 정확해야 정답으로 간주합니다. 따라서 답변을 최대한 간결하게 써 주시기 바랍니다.

---

Time: June 10. (Mon) 9:00 - 11:00 (120 min.)
Place: N1 102 (same place as the lecture room)

- Open book but no digital devices allowed
- Write your answers in Korean if you are a native Korean student; English otherwise
- No questions are allowed during the exam. If you need to make assumptions, please make sure to mention the assumptions.
- Score is applied in And operation: For each problem, you will only get points if every part of your answer is correct. Thus, please make your answers brief.

**Comment by goodtaeeun** [comment]
시험 시간이 수정되었습니다. (100분 -> 120분)

---

Exam time is updated. (100 min. -> 120 min.)

**Comment by leporia** [comment]
As it is my first open book exam at KAIST what can we bring?

Is it fine to bring notes and other self made material or are only lecture slides allowed?

**Comment by goodtaeeun** [comment]
You may bring any printed/written material.
Just keep in mind that no electronic devices are allowed.
